io.openems.backend.common/src/io/openems/backend/common/metadata/Edge.java

```diff
@@ -25,7 +25,7 @@ public class Edge {
 	private String comment;
 	private SemanticVersion version;
 	private String producttype;
-	private ZonedDateTime lastmessage;
+	private ZonedDateTime lastmessage = null;
```

Comment:
Not needed to set it to null here. In java this is done by default (see [here](https://stackoverflow.com/questions/16699593/uninitialized-object-vs-object-initialized-to-null/16699632#16699632) )

Reply:
Just as a note for future PRs